src/Compilers/Core/Portable/DiagnosticAnalyzer/SuppressMessageInfo.cs

```diff
@@ -8,5 +8,6 @@ internal struct SuppressMessageInfo
         public string Scope;
         public string Target;
         public string MessageId;
+        public AttributeData Attribute;
```

Comment:
what is life time of this suppressmessageinfo? is attributedata self contained and no reference to any big data structure?